sys-tuner/src/main.rs

```diff
@@ -0,0 +1,62 @@
+use std::fs;
+use unix_socket::UnixListener;
+
+#[cfg(any(not(unix), target_os = "macos"))]
+pub const SOLANA_SYS_TUNER_PATH: &str = "/tmp/solana-sys-tuner";
+
+// Use abstract sockets for Unix builds
+#[cfg(all(unix, target_os = "linux"))]
+pub const SOLANA_SYS_TUNER_PATH: &str = "\0/tmp/solana-sys-tuner";
+
+#[allow(dead_code)]
+#[cfg(target_os = "linux")]
+fn tune_system() {
+    use std::process::Command;
+    use std::str::from_utf8;
+    use thread_priority::*;
+
+    let output = Command::new("ps")
+        .arg("-eT")
+        .output()
+        .expect("Expected to see all threads");
+
+    if output.status.success() {
+        if let Ok(threads) = from_utf8(&output.stdout) {
+            let mut threads: Vec<&str> = threads.split('\n').collect();
+            for t in threads.iter_mut() {
+                if t.find("solana-poh-ser").is_some() {
+                    let pids: Vec<&str> = t.split_whitespace().collect();
+                    let thread_id = pids[1].parse::<u64>().unwrap();
+                    let _ignored = set_thread_priority(
+                        thread_id,
+                        ThreadPriority::Max,
+                        ThreadSchedulePolicy::Realtime(RealtimeThreadSchedulePolicy::Fifo),
+                    );
+                    break;
+                }
+            }
+        }
+    }
+}
+
+#[allow(dead_code)]
+#[cfg(any(not(unix), target_os = "macos"))]
+fn tune_system() {}
+
+#[allow(dead_code)]
```

Comment:
What's all the `#[allow(dead_code)]` about?

Reply:
The compiler was complaining about it, since it's getting built for lib as well.